internal/p2p/channel.go

```diff
@@ -214,3 +219,135 @@ func MergedChannelIterator(ctx context.Context, chs ...Channel) *ChannelIterator
 
 	return iter
 }
+
+////////////////////////////////////////////////////////////////////////
+
+type libp2pChannelImpl struct {
+	chDesc  *ChannelDescriptor
+	ps      *pubsub.PubSub
+	h       host.Host
```

Comment:
These are used infrequently enough and are peculiar enough to the implementation that I think they could be given full names `pubsub` and `host`.